vividus-plugin-json/src/main/java/org/vividus/json/steps/JsonSteps.java

```diff
@@ -74,16 +75,19 @@
     private final JsonUtils jsonUtils;
     private final ISoftAssert softAssert;
     private final IAttachmentPublisher attachmentPublisher;
+    private final Map<String, Matcher<Object>> customJsonMatchers;
 
     public JsonSteps(FluentEnumConverter fluentEnumConverter, JsonContext jsonContext, VariableContext variableContext,
-            JsonUtils jsonUtils, ISoftAssert softAssert, IAttachmentPublisher attachmentPublisher)
+            JsonUtils jsonUtils, ISoftAssert softAssert, IAttachmentPublisher attachmentPublisher,
+                     Map<String, Matcher<Object>> customJsonMatchers)
```

Comment:
or replace with setters approach?

Reply:
nope, injecting via constructor is an appropriate way